openedx/core/djangoapps/content/course_metadata/tasks.py

```diff
@@ -0,0 +1,33 @@
+"""
+This module has implementation of celeary tasks to compute course aggregate metadata
+"""
+
+import logging
+
+from celery.task import task
+from opaque_keys.edx.keys import CourseKey
+from .models import CourseAggregatedMetaData
```

Comment:
I can't remember if relative imports are acceptable anymore, from a quality standpoint -- I think it would be better to spell them out.  Also local imports should be the last set of imports in the list